shared/chat/selectable-big-team-channel-container.js

```diff
@@ -1,28 +1,40 @@
 // @flow
 import * as Constants from '../constants/chat2'
+import * as Styles from '../styles'
 import SelectableBigTeamChannel from './selectable-big-team-channel'
 import * as Types from '../constants/types/chat2'
 import {namedConnect} from '../util/container'
 
 type OwnProps = {|
   conversationIDKey: Types.ConversationIDKey,
   isSelected: boolean,
+  maxSearchHits?: number,
+  numSearchHits?: number,
   onSelectConversation: () => void,
 |}
 
 const mapStateToProps = (state, {conversationIDKey}) => {
+  const showBold = Constants.getHasUnread(state, conversationIDKey)
+  const showBadge = Constants.getHasBadge(state, conversationIDKey)
   const {channelname, teamname} = Constants.getMeta(state, conversationIDKey)
-  return {channelname, teamname}
+  return {channelname, showBadge, showBold, teamname}
 }
 
 const mapDispatchToProps = () => ({})
 
-const mergeProps = (stateProps, dispatchProps, ownProps) => ({
-  channelname: stateProps.channelname,
-  isSelected: ownProps.isSelected,
-  onSelectConversation: ownProps.onSelectConversation,
-  teamname: stateProps.teamname,
-})
+const mergeProps = (stateProps, dispatchProps, ownProps) => {
+  const isSelected = ownProps.isSelected && !Styles.isMobile
```

Comment:
I'm curious about the reason for this change. We need the background style specifically for mobile. What's the use case for chat search that needed this override?

Reply:
What do you even use this for?

Reply:
What do you even use this for?

Reply:
This is used in sendAttachmentInChat, where user chooses a conversation, and hit a Send button on upper right. It's currently feature flagged off and we're aiming to turn it on for this release. This is the only blocker for graduating that now.